### DISCUSSION TITLE
Community Discord For Help/Assistance?

### DISCUSSION THREAD

**Discussion by JailbreakPapa** [jrouwe/JoltPhysics#212] [Ideas]
Since This Physics Engine is kind of fresh at the moment for many developers, i feel like a central place that people could ask questions about the physics engine, and report bugs and such, in a quicker fashion.

**Comment by jrouwe** [comment]
Hello, sorry for the slow reply (was away). Personally, I would prefer using the issue tracker / the discussion forums here on github since it keeps every subject nicely separated and searchable. On Discord, I often see many discussions happening at the same time making it hard to find relevant information when you're trying to find relevant information later. But if there are many people who want this then I can make a channel for it.
